app/styles/modules/_settings.scss

```diff
@@ -573,3 +573,49 @@ section.modal-panel {
     margin-bottom: 30px;
   }
 }
+
+.fxa-radio {
```

Comment:
you probably do not need this `fxa-radio` class. Instead use `input[type='radio']`